spinalcordtoolbox/vertebrae/core.py

```diff
@@ -65,9 +74,10 @@ def vertebral_detection(fname, fname_seg, contrast, param, init_disc, verbose=1,
     sct.printv('Path template: ' + path_template, verbose)
 
     # adjust file names if MNI-Poly-AMU template is used (by default: PAM50)
-    fname_level = get_file_label(os.path.join(path_template, 'template'), id_label=7, output='filewithpath')  # label = spinal cord mask with discrete vertebral levels
-    id_label_dct = {'T1': 0, 'T2': 1, 'T2S': 2}
```

Comment:
I don't know if it was removed but we used to have like T1 and T2s image (same as 1: PAM50_small_t2) with other modalities, I believe it was only used here so it might have been removed :)